### PR TITLE
Update user workflow

### PR DESCRIPTION
**Pull request by not4Pedro**
Update the End To End user workflow as requested

### CODE REVIEW

File: content/en/docs/user-stories/user-workflow.md [hunk 1]
```diff
@@ -6,17 +6,17 @@ weight: 15
 
 Upon logging in, the analyst is greeted by the Dashboard which prominently displays the most significant tags, providing a quick glimpse into the trending topics and themes. This view sets the context for the analyst's tasks ahead.
 
-Next, the analyst navigates to the Analyze tab to initiate a new report, carefully setting its metadata to ensure proper categorization and tracking. This setup is crucial for the coherence and retrieval of the report later in the workflow.
```
Comment (not4Pedro) [inline content/en/docs/user-stories/user-workflow.md:9]: Maybe this makes sense to keep?

File: content/en/docs/user-stories/user-workflow.md [hunk 2]
```diff
@@ -6,17 +6,17 @@ weight: 15
...
+# Assess
+For easier filtering, in the user setting it is possible to set the end time of a shift.
```
Comment (not4Pedro) [inline content/en/docs/user-stories/user-workflow.md:10]: This line wasn't requested, but doesn't it make sense to use it here as well?

File: content/en/docs/user-stories/user-workflow.md [hunk 3]
```diff
@@ -6,17 +6,17 @@ weight: 15
...
-
-During the review process in the Assess tab, the analyst has several tools at their disposal to optimize their workflow:
-
-Voting on Stories: The analyst can vote stories up or down, and mark them as important using buttons or keyboard shortcuts (Ctrl+I), to highlight stories of particular relevance.
-Mark as Read: To avoid redundancy and ensure efficiency, the analyst can mark stories as read (or unread if needed) with a button or a shortcut (Ctrl+Space).
-Tags: By applying tag filters, the analyst can narrow down the selection to the most pertinent articles, enhancing focus and efficiency.
-If a story needs editing or creation, the analyst can do so directly within the Assess tab, where each news item can be adjusted until it is locked into a report. This capability is vital for maintaining the freshness and accuracy of the content.
-
-Once the relevant stories are collected and reviewed, the analyst returns to the Analyze tab to finalize the report, marking it as completed. This marks the transition from collection and review to dissemination.
```
Comment (not4Pedro) [inline content/en/docs/user-stories/user-workflow.md:20]: This section is interesting, maybe we could keep something from it.

Comment (b3n4kh) [reply content/en/docs/user-stories/user-workflow.md:20]: Yeah maybe just under a:

### additional tasks during assess

subsection?

Comment (not4Pedro) [reply content/en/docs/user-stories/user-workflow.md:20]: sounds good

**Pull request merged**